src/components/homepage/featuredWork.js

```diff
@@ -17,6 +17,12 @@ const FeaturedWorkSection = styled.section`
   }
 `
 
+const FeaturedWorkBox = styled.div`
+  max-width: 1290px;
```

Comment:
This is a wild 2x nitpick, but elsewhere on the site we have the max-width for the section as 1288px so we might as well stick with that so they're all the same. IDEALLY these are variables so you don't have to guess but this site's CSS game is a bit chaotic.

Reply:
Switch the max-width to 1288px;